docs/guide/http_cache.md

```diff
@@ -292,14 +292,14 @@ ezpublish:
 
     Both `purge_type` and `purge_server` can be set in one of the following ways:
 
-    - in `app/config/ezplatform.yml`
-    - by adding the parameter `purge_type` or `purge_server` respectively in `app/config/parameters.yml`
+    - in `config/packages/ezplatform.yaml`
+    - by adding the parameter `purge_type` or `purge_server` respectively in `config/packages/ezplatform.yaml`
     - by setting the `HTTPCACHE_PURGE_TYPE` environment variable.
 
-    It is recommended to use either `app/config/parameters.yml` or the environment variable.
+    It is recommended to use either `config/services.yaml` or the environment variable.
 
-    Note that in `app/config/ezplatform.yml`, the `purge_server` setting is an array while `purge_server` in
-    `app/config/parameters.yml` and the `HTTPCACHE_PURGE_SERVER` environment variable should be a string.
+    Note that in `config/ezplatform.yaml`, the `purge_server` setting is an array while `purge_server` in
+    `config/parameters.yaml` and the `HTTPCACHE_PURGE_SERVER` environment variable should be a string.
```

Comment:
There is no `config/parameters.yaml` file. Users should define parameters in `config/services.yaml` or better in `config/packages/package_file.yaml`.